transmogrifier/sources/dspace_dim.py

```diff
@@ -0,0 +1,276 @@
+        citation_creators = []
+        for creator in [f for f in all_fields if f.attrs["element"] == "creator"]:
+            creator_name = creator.string
+            citation_creators.append(creator_name)
+            c = Contributor(
+                value=creator_name,
+                kind="Creator",
+            )
+            kwargs.setdefault("contributors", []).append(c)
+
+        for contributor in [
+            f for f in all_fields if f.attrs["element"] == "contributor"
+        ]:
+            c = Contributor(
+                value=contributor.string,
+            )
+            if "qualifier" in contributor.attrs:
+                c.kind = contributor.attrs["qualifier"]
+            kwargs.setdefault("contributors", []).append(c)
+
+        # dates
+        date_issued = ""
+        for date in [f for f in all_fields if f.attrs["element"] == "date"]:
+            d = Date(value=date.string)
+            if "qualifier" in date.attrs:
+                d.kind = date.attrs["qualifier"]
+            if date.attrs["qualifier"] == "issued":
+                date_issued = date.string
+            kwargs.setdefault("dates", []).append(d)
+
+        # file_formats
+        for file_format in [
+            f
+            for f in all_fields
+            if f.attrs["element"] == "format"
+            and "qualifier" in f.attrs
+            and f.attrs["qualifier"] == "mimetype"
+        ]:
+            kwargs.setdefault("file_formats", []).append(file_format.string)
+
+        # format
+        kwargs["format"] = "electronic resource"
+
+        # funding_information
+        for funding_reference in [
+            f
+            for f in all_fields
+            if f.attrs["element"] == "description"
+            and "qualifier" in f.attrs
+            and f.attrs["qualifier"] == "sponsorship"
+        ]:
+            fr = Funder(
+                funder_name=funding_reference.string,
+            )
+            kwargs.setdefault("funding_information", []).append(fr)
+
+        # identifiers
+        identifiers = [f for f in all_fields if f.attrs["element"] == "identifier"]
+        for identifier in identifiers:
+            if "qualifier" not in identifier.attrs:
+                i = Identifier(
+                    value=identifier.string,
+                )
+                kwargs.setdefault("identifiers", []).append(i)
+            elif (
+                "qualifier" in identifier.attrs
+                and identifier.attrs["qualifier"] != "citation"
+            ):
+                i = Identifier(
+                    value=identifier.string,
+                    kind=identifier.attrs["qualifier"],
+                )
+                kwargs.setdefault("identifiers", []).append(i)
+
+        # language
+        for language in [
+            f
+            for f in all_fields
+            if f.attrs["element"] == "language"
+            and "qualifier" in f.attrs
+            and f.attrs["qualifier"] == "iso"
```

Comment:
Is there a reason we're explicitly restricting to languages with the "iso" qualifier?